### DISCUSSION TITLE
v9 to WebComopnents

### DISCUSSION THREAD

**Discussion by psuplat** [microsoft/fluentui#26542] [General] [answered]
I am aware that majority of dev resources are (probably) right now committed to getting the Fluent UI React v9 done according to the roadmap, but I am just curious if their are any plans of upgrading the WebComponents to match the v9, style & functionality wise, so  that they can be utilised in other frameworks, like vue or angular?

**Comment by layershifter** [comment, marked as answer]
@psuplat yes, there is an effort for `@fluentui/web-components@3`. You can follow the progress on this brunch, https://github.com/microsoft/fluentui/tree/web-components-v3.

**Reply by carl-egge** [reply]
Hi,
do you have any information when the web components implementation of Fluent 2 will be released as npm package? Is there some kind of maintained roadmap where we can see whether the implementation goes to production in a few weeks or rather a few years?

Since the branch seems to be in beta.6 it is probably feature-complete. Are there any documented steps on how to use the beta version in a project?

Thanks in advance for any help!

**Reply by khmakoto** [reply]
@chrisdholt can probably share more information about roadmap + steps to use the beta.
